src/vl-select.js

```diff
@@ -222,8 +242,15 @@ export class VlSelect extends vlFormValidationElement(nativeVlElement(HTMLSelect
    * @param {Object} params - object with callbackFn: function(select) with return value the items for `setChoices`
    * @fires VlSelect#VlSelectReady ready event wordt verstuurd wanneer veilige interactie met de webcomponent mogelijk is.
    */
+
   dress(params) {
     setTimeout(() => {
+      if (this.getAttribute('data-vl-search-placeholder')) {
+        this.searchPlaceholderTranslation = this.getAttribute('data-vl-search-placeholder');
+      } else {
+        this.searchPlaceholderTranslation = VlSelect.DEFAULT_SEARCH_PLACEHOLDER;
+      }
```

Comment:
Best uitbreiden en verhuizen naar aparte functie (zoals bijvoorbeeld) `setTranslations`.

Reply:
Ik had voor dit ticket enkel de data-vl-seach-placeholder gedaan omdat de andere 3 niet echt heel duidelijk waren. Het lukte me ook enkel om in de demo de tekst van de select.search_placeholder_value en de tekst van select.no_results zichtbaar te maken.

Een placeholder zetten wij via een placeholder attribuut op een option-element. Daarmee dat ik select.placeholder_value niet gebruikt heb. Je vindt dat terug in onze demo pagina en in de demopagina van AIV. Het lukte me ook niet om de default placeholdervalue zichtbaar te krijgen. 

Het lukte me ook niet om iets zichtbaar te krijgen wanneer de select.no_results vertaling gezet werd. Bovendien zie ik dat AIV een attribuut heeft ingevoerd 'data-vl-select-search-empty-text' om de default text waarnaar select.no_results wordt omgezet te tonen. 

De tekst die normaal bij select.no_more_options vond ik ook nergens terug in één of ander voorbeeld, of is mezelf ook niet gelukt om deze zichtbaar te maken.

Heb jij ideeën hoe ik testen kan schrijven voor de ontbrekende i18n variabelen (meer bepaald een fixture met 2 selects met verschillende waarden voor het desbetreffende attribuut)?

Reply:
@wautergu hieronder een screenshot van al de i18n teksten:

select.search_placeholder_value
<img width="2560" alt="select search_placeholder_value" src="https://user-images.githubusercontent.com/11139249/114530097-eb0ddf80-9c4a-11eb-8aff-cfca2a43b207.png">

select.placeholder_value
<img width="2560" alt="select placeholder_value" src="https://user-images.githubusercontent.com/11139249/114530135-f3661a80-9c4a-11eb-8028-e3de518ecd31.png">

select.no_results
<img width="2560" alt="select no_results" src="https://user-images.githubusercontent.com/11139249/114530182-fe20af80-9c4a-11eb-9a41-de20dba186ad.png">

select.no_more_options
<img width="2560" alt="select no_more_options" src="https://user-images.githubusercontent.com/11139249/114530205-04169080-9c4b-11eb-8248-1b88f379596b.png">